provisioning/common/etcd/values.yaml

```diff
@@ -1,15 +1,28 @@
-# See default values: https://github.com/bitnami/charts/blob/master/bitnami/etcd/values.yaml
-image:
-  tag: 3.5.5-debian-11-r16
+# Common etcd definition for the Templates API and Buffer API.
+# Values can be overridden by Helm --set flag or by another --values files.
+# Chart default values: https://github.com/bitnami/charts/blob/master/bitnami/etcd/values.yaml
```

Comment:
Vacsina nastaveni `etcd` je spolocna pre templates api aj buffer api.
Helm umoznuje prepisat jednotlive fields cet `--set` flag, alebo pridat nejaky dalsi doplnujuci yaml subor cez `--values` flag.

Takze som to presunul sem a doplnil komentare:
![image](https://user-images.githubusercontent.com/19371734/200869681-34c44884-7b67-4ec9-baf8-d909a119e7f6.png)